other-samples/java-sample/pom.xml

```diff
@@ -7,18 +7,29 @@
     <groupId>com.microsoft.cognitiveservice</groupId>
     <artifactId>anomalydetction</artifactId>
     <version>1.0</version>
-
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.0</version>
+                <configuration>
+                    <source>11</source>
+                    <target>11</target>
+                </configuration>
+            </plugin>
+        </plugins>
+    </build>
     <dependencies>
-        <dependency>
-            <groupId>org.apache.httpcomponents</groupId>
-            <artifactId>httpclient</artifactId>
-            <version>4.5.2</version>
-        </dependency>
         <dependency>
             <groupId>org.json</groupId>
```

Comment:
You don't need to pull in this dependency. The track2 libraries have a dependency on Jackson which will easily deserialize the json file to an object.

Reply:
According to the comment，fixed dependency package import settings in POM file